content/zh/docs/setup/install/external-controlplane/index.md

```diff
@@ -105,10 +105,10 @@ $ export REMOTE_CLUSTER_NAME=<your remote cluster name>
     istiod-68488cd797-mq8dn                1/1     Running   0          38s
     {{< /text >}}
 
-    您会注意到在 `istio-system` 名称空间中也创建了一个 Istiod 部署。 这用于配置 Ingress 网关，而不是远程集群使用的控制平面。
+    您会注意到在 `istio-system` 命名空间中也创建了一个 Istiod 部署。 这用于配置 Ingress 网关，而不是远程集群使用的控制平面。
 
     {{< tip >}}
-    可以将 Ingress 网关配置为在外部集群上的不同名称空间中承载多个外部控制平面，尽管在本示例中，您将仅在 `external-istiod` 名称空间中部署一个外部 Istiod。
+    可以将 Ingress 网关配置为在外部集群上的不同命名空间中承载多个外部控制平面，尽管在本示例中，您将仅在 `external-istiod` 命名空间中部署一个外部 Istiod。
     {{< /tip >}}
 
 1. 使用带有 TLS 的公共主机名配置您的环境来暴露 Istio Ingress 网关服务。 将 `EXTERNAL_ISTIOD_ADDR` 环境变量设置为主机名，将 `SSL_SECRET_NAME` 环境变量设置为包含 TLS 证书的密钥：
```

Comment:
The en source has changed 100+ lines from this line 114. Could you help to check through and add the zh text too?